generate_test.R

```diff
@@ -0,0 +1,14 @@
+
+# generate test_result and compare to analytical results
+# all temp increases are set to zero
+# temp pulses are set to zero except for Angola which gets a temperature pulse of 1 degree in 2021
+
+# default option is RCP 4.5, SSP1, eta of 1 and bhm damage function
+# change to option to choose different values? But how to do so within a source function
+source(generate_cscc_test.R)
+
+View(cscc)
+# test values from datatable cscc and wscc to say whether the test has passed]
+if (cscc$scc[2] < 1){
```

Comment:
You should test that the *magnitude* is less than one (because it could be huge and negative). Is 1 really the best we can do? Can we have a fail notice too?

Reply:
Why are you only testing one value in the whole array when the comment indicates you are testing several arrays?

Reply:
I just put the 1 because I did not calculate a sensible value yet. But yes, it should definitely be a better number.